src/App.jsx

```diff
@@ -36,10 +36,11 @@ ErrorFallback.propTypes = {
 
 export default function App() {
   const app = useFirebaseApp();
-  useInitPerformance(async () => {
+  const analytics = useAnalytics();
+  useMemo(async () => {
```

Comment:
`useMemo` doesn't seem useful if you are not storing the result. Should this be changed to a `useEffect`?

Reply:
yeah that makes sense, i think it's guarding from calling it multiple times so an effect would work